task/buildah-rhtap/0.1/buildah-rhtap.yaml

```diff
@@ -106,10 +108,15 @@ spec:
       name: tmpfiles
 
   - name: merge-sboms
-    image: registry.access.redhat.com/ubi9/python-39:1-165@sha256:4da8ddb12096a31d8d50e58ea479ba2fe2f252f215fbaf5bf90923a1827463ba
+    image: registry.access.redhat.com/ubi8/python-311@sha256:8ded4b6d8087706b6819ddda5d31f22b80e5aa4efa772e94d750699ccfbf98eb
+    env:
+      - name: SBOM_BLOB_URL
+        value: $(results.SBOM_BLOB_URL.path)
```

Comment:
nitpick: the name should make it clear that this is the result path, not a url

Reply:
Changed it. SBOM_BLOB_URL_RESULT_PATH sounded to Javaish to me, so I left it only as RESULT_PATH since we only have a single one for this step.